tests/socialcontext_test.py

```diff
@@ -16,3 +24,57 @@ def test_convex_hull_vs_alpha_border():
         np.logical_not(alpha_border), convex_hull
     )
     assert not np.any(in_alpha_border_not_in_convex_hull)
+
+
+@pytest.mark.parametrize("num_neighbours", [1, 3, 15])
+def test_neighbour_indices_vs_adjacency_matrix(num_neighbours):
+    t = np.load(cons.test_raw_trajectories_path, allow_pickle=True)
+    tt.interpolate_nans(t)
+
+    nb_indices = neighbour_indices(t, num_neighbours=num_neighbours)
+    assert nb_indices.shape == tuple(
+        [t.shape[0], t.shape[1], num_neighbours + 1]
```

Comment:
can't you do `t.shape[:2] + (num_neighbours + 1, )`? I think it is shorter

Reply:
Alternatively, since you are using `t.shape[0]` and `t.shape[1]` two times in this test, maybe you want to create a variable.

Reply:
I ended up doing this last thing everywhere and it solved the other issues